modules/persistent-storage-local-discovery.adoc

```diff
@@ -36,20 +36,19 @@ Use the `LocalVolumeSet` object with caution. When you automatically provision p
 
 . To enable automatic discovery of local devices from the web console:
 
-.. In the _Administrator_ perspective, navigate to *Operators* -> *Installed Operators* and click on the *Local Volume Discovery* tab.
+.. Click *Operators* -> *Installed Operators*.
 
-.. Click *Create Local Volume Discovery*.
+.. In the `openshift-local-storage` namespace, click *Local Storage*.
```

Comment:
Maybe a separate step to navigate into the namespace?

Reply:
Clicking *Installed Operators* opens a window in which the list of operators and the namespace in which they are created is visible. It's just that the user has to click *Local Storage* for which the value of the namespace field is `openshift-local-storage`. So navigating the namespace is actually not required.